R/status.R

```diff
@@ -56,7 +56,7 @@ renv_status_impl <- function(project, library, lockpath) {
   renv_status_check_unknown_sources(project, lockfile)
   renv_status_check_cache(project)
 
-  list(library = libstate, lockfile = lockfile)
+  list(library = libstate, lockfile = lockfile, synchronized = synchronized)
```

Comment:
Now that my changes are internal to `load()`, I don't have to modify public interface. Let me know if it's preferred to plumb out synchronization status using internals only.

Reply:
That sounds good to me.